frontend/src/components/Modal/ProfileModal/ProfileModal.tsx

```diff
@@ -96,9 +191,25 @@ export const ProfileModal = ({
                     error={formik.errors[item.id]}
                 />
             ))}
-            <button type="submit" className={styles.submitButton}>
-                {isNew ? <span>Create new profile!</span> : <span>Update profile!</span>}
-            </button>
+            <div className={styles.profileButtons}>
+                {!isNew ? (
+                    <button
+                        type="button"
+                        className={styles.deleteButton}
+                        onClick={() => {
+                            changeVisiblity();
+                            openConfirmModal && openConfirmModal();
+                        }}
+                    >
+                        Delete
+                    </button>
+                ) : (
+                    <></>
```

Comment:
Suggestion: maybe you can use short circuit instead of returning <></> ?

Reply:
🥇